src/main/scala/fable/Consumer.scala

```diff
@@ -6,6 +6,7 @@ import cats.Monad
 import fs2.Stream
 import io.chrisdavenport.log4cats.{slf4j, Logger}
 import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.apache.kafka.clients.consumer.{Consumer => ConsumerInterface}
```

Comment:
I generally find import renaming confusing, because I need to remember more than one name for the same thing. I'd recommend importing a package, like `org.apache.kafka`, and then you can refer to it as `kafka.clients.consumer.Consumer`. That's still kind of verbose, but I think it's clearer.

Reply:
After trying it, it does look verbose, but also it's not clear that we're not working with a concrete class. We feel like it's worth somehow preserving the fact that it is an interface.

Reply:
I feel strongly about this one. I think there's a good argument to be made that Kafka's decision for naming wasn't great (`kafka.KafkaConsumer` vs `kafka.Consumer`), but I think introducing a third name adds confusion rather than resolving it.

If you feel strongly about the namespacing, an alternative would be to rename `fable.Consumer` to `fable.FableConsumer` or something like that, so that it doesn't clash with `Consumer` from kafka-clients when both are imported.